portal-web/test/functional/com/liferay/portalweb/macros/mymacros/MyMacro.macro

```diff
@@ -0,0 +1,37 @@
+definition {
+
+	macro assertScreenCustomValues {
+		AssertTextEquals(
+			locator1 = "MyPath#CUSTOM_SIZE_HEIGHT",
+			value1 = 1920);
+
+		AssertTextEquals(
+			locator1 = "MyPath#CUSTOM_SIZE_WIDTH",
+			value1 = 1080);
+	}
+
+	macro previewScreenCustomSize {
```

Comment:
Hey, the purpose of this macro is to define the values right? When you put the preview as the name of the macro you are not telling that the macro is defining some values, so I believe that you could try to change the name of the macro a little bit to match what it's actually doing.
You can try something like:

-  setScreenCustomSize
- addCustomSize
-  and it goes on...